tests/integration/test_cli.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+from unittest import mock
+
+from fastapi_security import cli
+
+
+def test_usage_output_without_params():
+    result = subprocess.run(["fastapi-security"], capture_output=True)
+    assert result.returncode == 2
+    assert result.stdout.decode().splitlines() == []
+    assert result.stderr.decode().splitlines() == [
+        "usage: fastapi-security [-h] {gendigest} ...",
+        "fastapi-security: error: the following arguments are required: subcommand",
+    ]
+
```

Comment:
To fix failing tests you could call cli.main directly as I've suggested below. Makes for faster tests as well.

Reply:
The failure was intentional, so to speak, but honestly I forgot about the pending change I intended to make. It is fixed now.

The problem is not in calling `cli.main` directly or not, but rather in that `add_subparsers` function only added `required=` kwarg in Python 3.7, so I wanted a test suite that would work with `required=True` in 3.7+ to then mimic the same behaviour (message and exit-code) for 3.6. See my most recent commit.

The reasons I went for subprocess approach in the first place are as follows:
- it does not require hard-coding the `prog=` kwarg
- it doubles as an integration test for the entire script (also testing that the entrypoint configuration is correct)
- it does not require catching a `SystemExit` exception to ensure tests continue working

If it were up to me, I'd probably keep the existing tests as is, and use `cli.main` as soon as there are more unit tests to add. Although if you insist I can change go for `cli.main` in this PR as well.

Reply:
@immerrr Gotcha. Let's drop support for 3.6, [it's reached end-of-life](https://devguide.python.org/devcycle/#maintenance-branches). You can include that change in your PR. Or if you'd rather I remove it and you rebase on top of my commit?

Reply:
Done (and reverted the change that enabled support for py3.6)

Reply:
Thanks @immerrr. Tests are still failing on Windows. And they also fail locally on my Mac OS. Fails to find command `fastapi-security`. I would suggest that you go the `cli.main` route.

Reply:
ok, will do